bundles/org.openhab.binding.mqtt.awtrixlight/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,593 @@
+		<description>How long the app should be displayed</description>
+		<category>time</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>Duration</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="effect">
+		<item-type>String</item-type>
+		<label>Background effect</label>
+		<description>Effect shown in the background of the app</description>
+		<category>screen</category>
+		<tags>
+			<tag>Control</tag>
+		</tags>
+		<state>
+			<options>
+				<option value="None">None</option>
+				<option value="BrickBreaker">BrickBreaker</option>
+				<option value="Fireworks">Fireworks</option>
+				<option value="Radar">Radar</option>
+				<option value="Snake">Snake</option>
+				<option value="TheaterChase">Theater Chase</option>
+				<option value="SwirlOut">Swirl Out</option>
+				<option value="LookingEyes">Looking Eyes</option>
+				<option value="Pacifica">Pacifica</option>
+				<option value="PlasmaCloud">Plasma Cloud</option>
+				<option value="Checkerboard">Checkerboard</option>
+				<option value="PingPong">Ping Pong</option>
+				<option value="Ripple">Ripple</option>
+				<option value="TwinklingStars">Twinkling Stars</option>
+				<option value="ColorWaves">Color Waves</option>
+				<option value="SwirlIn">Swirl In</option>
+				<option value="Matrix">Matrix</option>
+				<option value="Plasma">Plasma</option>
+				<option value="MovingLine">Moving Line</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="effectPalette">
+		<item-type>String</item-type>
+		<label>Effect Color Palette</label>
+		<description>Changes the color scheme of the effect settings</description>
+		<category>colorpicker</category>
+		<tags>
+			<tag>Control</tag>
+		</tags>
+		<state>
+			<options>
+				<option value="None">None</option>
+				<option value="Cloud">Cloud</option>
+				<option value="Lava">Lava</option>
+				<option value="Ocean">Ocean</option>
+				<option value="Forest">Forest</option>
+				<option value="Stripe">Stripe</option>
+				<option value="Party">Party</option>
+				<option value="Heat">Heat</option>
+				<option value="Rainbow">Rainbow</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="effectBlend" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Smooth Effects</label>
+		<description>Smoother effect animations</description>
+		<category>switch</category>
+		<tags>
+			<tag>Control</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="effectSpeed" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Background effect speed</label>
+		<description>Playback speed of background animations</description>
+		<category>time</category>
+		<tags>
+			<tag>Control</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="text">
+		<item-type>String</item-type>
+		<label>Display Text</label>
+		<description>Text displayed in the app</description>
+		<category>text</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>Duration</tag>
+		</tags>
```

Comment:
drop the tags, there is no duration being controlled here.

Reply:
Done